Classes/Hooks/UserAuthHook.php

```diff
@@ -90,6 +91,29 @@ protected function showForm($token)
 
         $backendExtConf = unserialize($this->getExtConf('backend'));
 
+        if (!empty($backendExtConf['loginBackgroundImage'])) {
+            $backgroundImage = $this->getUriForFileName($backendExtConf['loginBackgroundImage']);
+            $css = '@media (min-width: 768px){';
+            $css .= '.typo3-login-carousel-control.right, ';
+            $css .= '.typo3-login-carousel-control.left, ';
+            $css .= '.panel-login { border: 0; } ';
+            $css .= '.typo3-login { background-image: url("' . $backgroundImage . '"); }';
+            $css .= '}';
+            $documentTemplate->inDocStylesArray[] = $css;
+        }
+
+        if (!empty($backendExtConf['loginLogo'])) {
+            $logo = $backendExtConf['loginLogo'];
+        } else {
+            if (!empty($backendExtConf['loginHighlightColor'])) {
```

Comment:
I just copied that from \TYPO3\CMS\Backend\Controller\LoginController::main to be sure it does the same as the core (TYPO3 8.4)